core/src/main/java/com/graphhopper/GHRequest.java

```diff
@@ -40,6 +38,10 @@
     private boolean possibleToAdd = false;
     private Locale locale = Locale.US;
 
+    // List of preferred start (1st element) and arrival directions (all other). 
+    // Directions are north based azimuth (clockwise) in (0,2pi) or NaN for equal preference
+    private List<Double> preferredDirections;
```

Comment:
we can do this, but it would brake at lot of points the common Usage of GHRequest.addPoint()
e.g. https://github.com/graphhopper/graphhopper/blob/master/core/src/test/java/com/graphhopper/GraphHopperTest.java#L519